orttraining/orttraining/core/graph/gradient_schema_defs.cc

```diff
@@ -1440,6 +1440,50 @@ Example 4:
           "be false.",
           "T");
 
+  ONNX_CONTRIB_OPERATOR_SCHEMA(PassThrough)
+      .SetSupportLevel(OpSchema::SupportType::EXPERIMENTAL)
+      .SetDomain(kMSDomain)
+      .SinceVersion(1)
+      .SetDoc("Passes input to output as is with a given signal.")
+      .Input(
+          0,
+          "control_signal",
+          "Control signal to pass tensors",
+          "T1")
+      .Input(
+          1,
+          "input_tensor",
```

Comment:
```suggestion
          "input_tensors",
```
or just `X` like ONNX [Loop operator](https://github.com/onnx/onnx/blob/master/docs/Operators.md#inputs-2---).